hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenClaimAirdropTest.java

```diff
@@ -331,6 +333,355 @@ final Stream<DynamicTest> twoAirdropsNFTSandTwoClaims() {
                 getAccountBalance(BOB).hasTokenBalance(NON_FUNGIBLE_TOKEN, 2));
     }
 
+    @HapiTest
+    @DisplayName("token airdrop 10 FT and claim them")
```

Comment:
Need to follow the HapiTest guidelines Michael had in a document. This has more than 7 operations 😄